sunkit_image/tests/conftest.py

```diff
@@ -155,7 +155,6 @@ def granule_minimap3():
 
 
 @pytest.fixture(params=["array", "map"])
-@pytest.mark.remote_data()
```

Comment:
Unfortunately this might fail the CI as we have to make sure any tests that call these fixtures are marked with `@pytest.mark.remote_data()`

Reply:
I think I made sure that this is the case, but I can check again.